packages/transport-chrome/session-client.ts

```diff
@@ -60,13 +61,10 @@ export class CRSessionClient {
 
   private disconnect = () => {
     this.clientPort.removeEventListener('message', this.clientListener);
-    this.clientPort.addEventListener('message', (ev: MessageEvent<unknown>) => {
-      if (isTransportEvent(ev.data)) {
-        const { requestId } = ev.data;
-        this.clientPort.postMessage({ requestId, error: 'Connection closed' });
-      }
+    this.clientPort.postMessage({
+      error: errorToJson(new ConnectError('Connection closed', Code.Unavailable), undefined),
     });
-    this.clientPort.postMessage({ error: { code: 'unavailable', message: 'Connection closed' } });
+    this.clientPort.close();
```

Comment:
Why this change?

Reply:
using connecterror now in this script. the lines immediately above this are conveying a transport-level error that is equivalent to this

https://github.com/penumbra-zone/web/blob/0cc03146874e28e8fff14046e5ba04a084f1f533/packages/transport-chrome/session-client.ts#L62-L68

previously this also attached a handler that closed every subsequent request attempt. but, any existing transports are failed by the close message, and no subsequent transports can init.